packages/preview/src/browser/preview-handler.ts

```diff
@@ -31,15 +40,67 @@ export namespace RenderContentParams {
     }
 }
 
+/**
+ * A PreviewHandler manages the communication between text editor and preview widget including whether a preview shall be shown at all.
+ * 
+ * See {@link MarkdownPreviewHandler} for an example implementation.
+ */
 export interface PreviewHandler {
+    /**
+     * One or more classes which shall be applied to the preview widget icon.
+     */
     readonly iconClass?: string;
+    /**
+     * Indicates whether and with which priority (larger is better) this preview handler is responsible for the given resource identified by the given URI.
+     * 
+     * @param uri the URI identifying a resource.
+     * 
+     * @returns A number larger than 0 if the handler is applicable, 0 or a negative number otherwise.
```

Comment:
I believe we should be consistent with the casing of the comments, lowercase would be best (all docs in the pull-request should follow).

```suggestion
     * @returns a number larger than 0 if the handler is applicable, 0 or a negative number otherwise.
```

Reply:
Makes sense! However only for sentences following after tags like `@returns` and `@param` right? I would leave the other sentences upper cased.

Reply:
Correct!